File: web/desktop/users-app/macOS/DatabaseConfig.py
#!/usr/bin/env python3
# Same database for macOS
DB_CONFIG = {
    'host': 'localhost',
    'database': 'tigerex',
    'user': 'root'
}
# Connects to shared backend
def get_connection():
    return psycopg2.connect(**DB_CONFIG)

File: web/ios/users-app/Swift/DatabaseConfig.swift
import Foundation
class DatabaseConfig {
    static let DB_HOST = "localhost"
    static let DB_NAME = "tigerex"
    static let API_URL = "https://api.tigerex.com"
    // Same backend connection
    static func connect() -> Connection {
        return Connection(host: DB_HOST, database: DB_NAME)
    }
}
